### ISSUE TITLE
Music isn't indexed

### ISSUE THREAD

**Issue by sandinmyjoints** [ghenry22/Subsonic_Synology#35]
I installed this package on a new DS218+, and I have some music in /volume1/music. I added that path to Subsonic's media folders and enabled it, and told Subsonic to rescan media folders, but none of the music inside of it shows up -- Subsonic can't seem to find or index it. Any ideas what could be going on, or how I can troubleshoot it?

**Comment by whitakre** [comment]
Sounds similar to what I experienced on my DS116. My problem was permissions.

Check if user subsonic has access to the folder. Control Panel/User/subsonic/Edit/Permissions

**Comment by sandinmyjoints** [comment]
Thanks, the user did not have permissions. I have added them. No dice so far, even after clicking rescan, but I'll give it some more time...

**Comment by sandinmyjoints** [comment]
OK, working now!

**Comment by ghenry22** [comment]
yeh permissions can be a bit finicky and DSM updates on the Synology sometimes make changes to how things work as well.  Once you get it right though it tends to be set and forget.  Glad you got it working.

**Comment by tedfranklin** [comment]
I'm having the same problem with a spanking new DS918+; I've been pulling out my hair.  Maybe I should just chill and see if I get lucky like sandinmyjoints. LOL. I'm using "/volume1/Music"; I don't get a "Folder not found" error from the subsonic admin screen,  I've tried flipping various switches, e.g., giving the "subsonic" user R/W access to the Music file; making "subsonic" part of the Administrators group.   I have no idea what the password is for the subsonic account so I haven't touched that.  No DSM update today so that isn't likely to be the problem.  I greatly appreciate the work you have put into creating this package; it was a major reason I decided to get a Synology NAS.  So far, I'm stumped about how to diagnose this.
